### PR TITLE
Add back check for course_info params

### PR DESCRIPTION
**Pull request by noisecapella**
#### Pre-Flight checklist

- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
None

#### What's this PR do?
Fixes a build error

#### How should this be manually tested?
Run `npm run build:zips` with a collection of courses. It should pass.

### CODE REVIEW

**Comment by aliceriot** [comment]
I was able to run `build:zips` successfully without errors, and I was getting a similar (I think) error to the RC build error locally.

I guess I'm wondering why we need to do this though - do we need to also do some detective work to figure out why courses are missing this data?

File: site/layouts/partials/course_features.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{ if isset "course_info" .Params }}
```
Comment (aliceriot) [inline site/layouts/partials/course_features.html:1]: yea Carey's comment about the argument order is correct: https://gohugo.io/functions/isset/#readout

**Review by aliceriot — CHANGES_REQUESTED** [review]

File: site/layouts/partials/course_features.html (hunk 1 above)
Comment (gumaerc) [reply site/layouts/partials/course_features.html]: Yup, looks correct with the latest commit.  I'm running a test run myself with RC data -> `ocw-to-hugo`@release -> `hugo-course-publisher` at this branch.

**Comment by gumaerc** [comment]
> I was able to run `build:zips` successfully without errors, and I was getting a similar (I think) error to the RC build error locally.
> 
> I guess I'm wondering why we need to do this though - do we need to also do some detective work to figure out why courses are missing this data?

Since this is being called in a partial, this code will be called anywhere the partial is rendered with any context that is passed into it.  These partials are both called from `course/baseof.html`, so they will be passed the context of any markdown file that contains the Front Matter `type: course`, which is essentially every markdown file at the moment.  Most likely what doesn't have the information are all the specialty `layout`s being used like video, pdf, etc.  Either way, I don't think it's a great solution to put that data into every markdown file.  This is why I keep bringing up Data Templates, because it would simplify obtaining this information.

**Comment by pdpinch** [comment]
> This is why I keep bringing up Data Templates, because it would simplify obtaining this information.

Let's just focus on getting the build to work as is, before we start refactoring the data structure.

**Comment by gumaerc** [comment]
I don't think the error in the RC build here had to do with `build:zips`, as I don't even think that's being run at the moment on the RC and Prod build servers.  I believe the Caddy script is just running `import:ocw` against `open-learning-course-data-rc` or `open-learning-course-data-production` and then `npm run build`  I'm fairly certain the error Mark shared in Slack occurred during the Hugo build.

**Comment by noisecapella** [comment]
I saw an error when running `npm run build:zips` locally but not when running `npm run build` locally. But I only have 4 courses at the moment

**Comment by gumaerc** [comment]
> > This is why I keep bringing up Data Templates, because it would simplify obtaining this information.
> 
> Let's just focus on getting the build to work as is, before we start refactoring the data structure.

Agreed, I just thought it was relevant to the conversation.

**Comment by pdpinch** [comment]
@noisecapella @markbreedlove Carey is right that we shouldn't be running build:zips. The feature has been removed entirely for the time being.

**Comment by gumaerc** [comment]
> @noisecapella @markbreedlove Carey is right that we shouldn't be running build:zips. The feature has been removed entirely for the time being.

I don't see it being called anywhere in the Caddy script that @markbreedlove shared on Slack.  The issue here is running `npm run build` with every course.  I have a test run going with all courses that finished `ocw-to-hugo` conversion and is running through the Hugo build with George's changes here to verify that all is good, then we can be confident that there will be no problems with a new release.

**Comment by gumaerc** [comment]
This is the error @markbreedlove posted in Slack:

```
ERROR 2020/11/20 13:44:56 render of "page" failed: execute of template failed: template: course/course_section.html:20:7: executing "course/course_section.html" at <partial "mobile_course_info.html" .>: error calling partial: execute of template failed: template: partials/mobile_course_info.html:6:5: executing "partials/mobile_course_info.html" at <partial "chp_partial.html" (dict "partial" "topics.html" "context" .)>: error calling partial: execute of template failed: template: partials/chp_partial.html:6:5: executing "partials/chp_partial.html" at <partial $partial (dict "Params" .Params "args" $args)>: error calling partial: execute of template failed: template: partials/topics.html:7:9: executing "partials/topics.html" at <partial "topic.html" (dict "index" $index "topic" $topic "subtopics" $topic.subtopics)>: error calling partial: "/opt/ocw/hugo-course-publisher/site/layouts/partials/topic.html:4:49": execute of template failed: template: partials/topic.html:4:49: executing "partials/topic.html" at <len .subtopics>: error calling len: len of untyped nil
```

The end of that error is showing that it was actually caused by `subtopics` not being set on a topic, causing Hugo to call `len` on `nil`.

**Comment by gumaerc** [comment]
My full site build succeed here, running against RC data.

**Review by gumaerc — APPROVED** [review]
:+1:

**Pull request merged**